lambda/index.js

```diff
@@ -0,0 +1,41 @@
+const AWS = require('aws-sdk')
```

Comment:
That's fair. Clearly I need to clean up the build process. `node-lambda` is nice, but it has been causing too many problems here.

I hate python. Any python code I write will be utter trash.

Reply:
I do prefer node but python's not all bad. I'm not sure what you mean, @cdaniluk, by python requiring external dependencies. The reason that I was given for standardizing on python was that it was a lingua franca for our team but as I've been working in python it's become apparent that isn't the case because the only person who's willing to review python PRs is Cris.

I really don't care what we standardize on if we do standardize now. If I had to pick something I would probably say that we should start investing in go.

Reply:
No. Go is way more complex and low level than anything we have a need for, and the community is super toxic. If we had a use for such a low-level language I'd vote for Rust, but we don't.

Reply:
I have yet to see evidence for either of those statements. Rust is too low, Go has a garbage collector and pointers aren't that difficult to understand